hole/play.go

```diff
@@ -315,6 +316,15 @@ func play(
 		// is not nil. This seems to be a quirk of the Babashka interpreter
 		// that only occurs when providing code via a command line argument.
 		code += "(print)"
+	case "jq":
+		if hole.ID == "quine" {
+			// Disable implicit output by rejecting input that can be parsed as JSON.
+			var js json.RawMessage
+			if err := json.Unmarshal([]byte(code), &js); err == nil {
```

Comment:
Something like https://pkg.go.dev/encoding/json#Valid would be simpler than parsing into a buffer that we throw away.

Reply:
```go
case "jq":
	if hole.ID == "quine" {
		// Disable implicit output by rejecting input that can be parsed as JSON.
		if json.Valid([]byte(code)) {
			return nil
		}
	}
```